R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -169,48 +169,50 @@ test_that("create DataFrame from RDD", {
   error = function(err) {
     skip("Hive is not build with SparkSQL, skipped")
   })
-  sql(hiveCtx, "CREATE TABLE people (name string, age double, height float)")
-  df <- read.df(hiveCtx, jsonPathNa, "json", schema)
+  assign(".sparkRHivesc", hiveCtx, envir = .sparkREnv)
```

Comment:
minor nit: we should add a new method to create the test hive context that also does this assignment. seems like something that other test cases might forget to do